### PR TITLE
Add images job to cluster-cloud-controller-manager-operator

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-master.yaml [hunk 1]
```diff
@@ -3,12 +3,23 @@ base_images:
...
     name: release
     namespace: openshift
     tag: golang-1.15
+images:
+- from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
+  to: cluster-cloud-contrller-manager-operator
```
Comment (Danil-Grigorev) [inline ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-master.yaml:22]: This name is not right. It also should be simply `cluster-cloud-controller-manager`, as is in https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/manifests/image-references#L8

Comment (Fedosin) [reply ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-master.yaml:22]: I think it should be called `cluster-cloud-contrller-manager-operator` to comply with other components

**Review by Danil-Grigorev — APPROVED** [review]
/lgtm

**Pull request merged**